Doc/library/importlib.metadata.rst

```diff
@@ -32,17 +32,17 @@ Overview
 
 Let's say you wanted to get the version string for a package you've installed
 using ``pip``.  We start by creating a virtual environment and installing
-something into it::
+something into it:
 
-.. highlight:: none
+.. code-block:: shell-session
 
     $ python3 -m venv example
     $ source example/bin/activate
     (example) $ pip install wheel
 
-You can get the version string for ``wheel`` by running the following::
+You can get the version string for ``wheel`` by running the following:
 
-.. highlight:: none
+.. code-block:: pycon
```

Comment:
s/pycon/python

Reply:
nah, pycon is correct "python console"

there's other examples in the docs:

```console
$ git grep ':: pycon' -- '*.rst'
Doc/extending/embedding.rst:.. code-block:: pycon
Doc/extending/extending.rst:.. code-block:: pycon
Doc/extending/newtypes_tutorial.rst:.. code-block:: pycon
Doc/extending/newtypes_tutorial.rst:.. code-block:: pycon
Doc/extending/newtypes_tutorial.rst:.. code-block:: pycon
Doc/extending/newtypes_tutorial.rst:.. code-block:: pycon
Doc/extending/newtypes_tutorial.rst:.. code-block:: pycon
Doc/faq/extending.rst:.. code-block:: pycon
Doc/faq/extending.rst:.. code-block:: pycon
Doc/faq/windows.rst:.. code-block:: pycon
Doc/faq/windows.rst:.. code-block:: pycon
Doc/howto/logging-cookbook.rst:.. code-block:: pycon
Doc/howto/logging-cookbook.rst:.. code-block:: pycon
Doc/install/index.rst:.. code-block:: pycon
Doc/library/importlib.metadata.rst:.. code-block:: pycon
Doc/library/unittest.mock.rst:.. code-block:: pycon
Doc/library/unittest.mock.rst:.. code-block:: pycon
```